dallinger/db.py

```diff
@@ -134,16 +134,24 @@ def wrapper(*args, **kw):
 # Reset outbox when session begins
 @event.listens_for(Session, 'after_begin')
 def after_begin(session, transaction, connection):
+    logger.debug(
+            'Clearing message queue due to begin: {}'.format(session.info))
```

Comment:
Playing in debug mode just now, I see that this is being called about 10x/second. Not sure what's triggering this(?)
```
14:59:46 web.1      |  2018-03-26 14:59:46,038 Clearing message queue due to begin: {'outbox': []}
14:59:46 web.1      |  2018-03-26 14:59:46,086 Clearing message queue due to begin: {'outbox': []}
14:59:46 web.1      |  2018-03-26 14:59:46,234 Clearing message queue due to begin: {'outbox': []}
14:59:46 web.1      |  2018-03-26 14:59:46,288 Clearing message queue due to begin: {'outbox': []}
14:59:46 web.1      |  2018-03-26 14:59:46,356 Clearing message queue due to begin: {'outbox': []}
14:59:46 web.1      |  2018-03-26 14:59:46,419 Clearing message queue due to begin: {'outbox': []}
14:59:46 web.1      |  2018-03-26 14:59:46,473 Clearing message queue due to begin: {'outbox': []}
14:59:46 web.1      |  2018-03-26 14:59:46,565 Clearing message queue due to begin: {'outbox': []}
14:59:46 web.1      |  2018-03-26 14:59:46,671 Clearing message queue due to begin: {'outbox': []}
14:59:46 web.1      |  2018-03-26 14:59:46,741 Clearing message queue due to begin: {'outbox': []}
14:59:46 web.1      |  2018-03-26 14:59:46,817 Clearing message queue due to begin: {'outbox': []}
14:59:46 web.1      |  2018-03-26 14:59:46,880 Clearing message queue due to begin: {'outbox': []}
14:59:46 web.1      |  2018-03-26 14:59:46,925 Clearing message queue due to begin: {'outbox': []}
14:59:46 web.1      |  2018-03-26 14:59:46,987 Clearing message queue due to begin: {'outbox': []}
```

Reply:
Yeah, I got this too, it's much more than I expected. I mean, it's the right use of the debug log level in that it's useful for debugging only, but we use debug quite a lot in real experiments.